bot/cogs/tags.py

```diff
@@ -28,21 +29,31 @@ class Tags(Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
         self.tag_cooldowns = {}
-
         self._cache = {}
-        self._last_fetch: float = 0.0
 
-    async def _get_tags(self, is_forced: bool = False) -> None:
+    @Cog.listener()
+    async def on_ready(self) -> None:
+        """Runs the code before the bot has connected."""
+        await self.get_tags()
+
+    async def get_tags(self) -> None:
```

Comment:
Since we are no longer doing any API call and simply reading from local files, the function `get_tags()` is no longer needed to be `async` and so you can do this
```suggestion
        self.get_tags()

    def get_tags(self) -> None:
```

Reply:
Also this event **can be** fired multiple times resulting in unnecessary blocking reading, so you might want to add it to the `setup()` function instead.